text/1522-conservative-impl-trait.md

```diff
@@ -471,7 +471,7 @@ trait Foo {
 
 Given the above code, there is no way for `bar` to choose a return type `X`
 that could fundamentally differ between instantiations of `Self`
-while still being instantiable with an arbitrary `U`.
```

Comment:
I'll definitely listen to the popular consensus here (because programming words), but the base word is `instantiate`; adding the `-able` suffix seems to produce `instantiatable`. Neither are liked by my spell checker, however.